aQute.libg/test/aQute/lib/io/IOTest.java

```diff
@@ -317,8 +344,10 @@ public void testCreateDirectory_Symlink() throws Exception {
 			.exists());
 	}
 
-	public void testCreateDirectory_SymlinkMissingTarget() throws Exception {
-		Path rootDirectory = Paths.get("generated/tmp/test/" + getName());
+	@Test
+	@DisabledOnOs(WINDOWS)
```

Comment:
Why did you disable this on Windows? This test currently runs on Windows.

Reply:
Ditto.